src/main/java/redis/clients/jedis/params/XPendingParams.java

```diff
@@ -52,6 +78,13 @@ public XPendingParams consumer(String consumer) {
 
   @Override
   public byte[][] getByteParams() {
+    if (start != null && startEndpoint == null) {
+      startEndpoint = RangeEndpoint.of(RangeEndpoint.convert(start));
```

Comment:
If anyway you're going to convert `start` to `startEndpoint` why not do it in the setter?